src/layouts/hass-tabs-subpage-data-table.ts

```diff
@@ -0,0 +1,124 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  query,
+  TemplateResult,
+} from "lit-element";
+import "../components/data-table/ha-data-table";
+// tslint:disable-next-line
+import {
+  HaDataTable,
+  DataTableColumnContainer,
+  DataTableRowData,
+} from "../components/data-table/ha-data-table";
+import "./hass-tabs-subpage";
+import { HomeAssistant, Route } from "../types";
+// tslint:disable-next-line
+import { PageNavigation } from "./hass-tabs-subpage";
+
+@customElement("hass-tabs-subpage-data-table")
+export class HaTabsSubpageDataTable extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public isWide!: boolean;
+  @property({ type: Boolean, reflect: true }) public narrow!: boolean;
+  @property({ type: Object }) public columns: DataTableColumnContainer = {};
+  @property({ type: Array }) public data: DataTableRowData[] = [];
+  @property({ type: Boolean }) public selectable = false;
+  @property({ type: String }) public id = "id";
```

Comment:
Can we start adding comments above each property to describe what they do. Use JS Doc standard.